index.html

```diff
@@ -4545,7 +4545,20 @@ <h2>
                 <li>Set <var>request</var>.<a>[[\state]]</a> to
                 "<a>closed</a>".
                 </li>
-                <li>Reject the promise
+                <li>If <var>request</var><a>[[\response]]</a> is not null,
```

Comment:
Missing dot

Reply:
D'oh... keep doing this. Need to add a linter rule for ReSpec ☺️